pkg/builtin/manifest.yml

```diff
@@ -1584,3 +1584,57 @@ extensions:
             - id: layerCamera
               title: Camera position
               type: camera
+  - id: cluster
+    name: Cluster
+    type: cluster
+    description: clustering...
```

Comment:
Describe the clustering function in more detail

Also each fields in cluster group should be described

Reply:
@issmail-basel FYI